onnxscript/optimizer/_constant_folding.py

```diff
@@ -891,12 +892,22 @@ def new_constant(self, irvalue: ir.Value, value):
         irvalue.const_value = tensor
 
         if value.nbytes > self._output_size_limit:
-            logger.info(
-                "Skip storing constant folded nvalue %s due to large size %s.",
-                irvalue.name,
-                value.nbytes,
-            )
-            return None
+            # Handle examples like Transpose(weight) to be folded even if the size is large,
+            # as long as weight has no other uses. This won't increase model size.
```

Comment:
Do we consider the computation cost for different ops?

Reply:
I understand the primary reasons for us to avoid processing large tensors are (1) compute cost and (2) potential increment in data size. This seems to address (2) but (1) can still be an issue, which can causes optimization to take a long time to finish

Reply:
Discussed offline. Currently, users can use input-size-limits to prevent processing large tensors. Several refinements/extensions are possible for this, but that's future work.